Makefile

```diff
@@ -12,12 +12,6 @@ BASE_DIR=$(CURDIR)
 
 TAG := # make sure tag is never injectable as an env var
 
-ifdef CI
```

Comment:
I think it's ok if we just set the `tag` to the value below, especially since this PR changes `CIRCLE_TAG` to be the output of `make tag` anyway